models/topics_model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+const data = require("../db/data/development-data/index");
+
+const allTopics = (order_by = "slug") => {
+  const acceptedTopics = ["description", "slug"];
```

Comment:
We don't need to worry about having queries for this task, so these bits can be removed. We're just interested in getting all topics for this ticket 🙂